app/src/test/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharacterPresenterTest.kt

```diff
@@ -70,7 +72,7 @@ class CharacterPresenterTest {
     @Ignore
```

Comment:
why this is ignored? 🤔

Reply:
It's a default code we had when i downloaded the code. Then when i gone to do the test i will changed